1.20.5/1.20.5-pre2.md

```diff
@@ -0,0 +1,15 @@
+advancement | `Snip it!` advancement is renamed to `Shear Brilliance`
+
+data | Using ctrl+pick block no longer lists "(+NBT)" in the item tooltip
+
+assets | The animated Nether Portal texture is displayed when changing dimension to or from The Nether
```

Comment:
Everything should be lowercase (also in other places)

Reply:
fixed.